docs/document/content/user-manual/shardingsphere-proxy/startup/graalvm-native-image.en.md

```diff
@@ -11,7 +11,21 @@ corresponding `Docker Image` through the `native-image` component of `GraalVM`.
 ## Notice
 
 - ShardingSphere Proxy is not yet ready to integrate with GraalVM Native Image.
-  It has daily build tasks at https://github.com/apache/shardingsphere/actions/ for testing builds.
+  It exists nightly builds at https://github.com/apache/shardingsphere/pkgs/container/shardingsphere-proxy-native.
+  Assuming there is a `conf` folder containing `server.yaml` as `./custom/conf`, you can test it with the
+  following `docker-compose.yml` file.
+
+````yaml
+version: "3.8"
+
+services:
+  apache-shardingsphere-proxy-native:
+    image: ghcr.io/apache/shardingsphere-proxy-native:latest
```

Comment:
The issue that this Docker Image cannot be viewed is tracked at https://github.com/orgs/community/discussions/35042 and it looks like there is an issue with the GitHub Package.